test/run_pass/test_buffer_from.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2015-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var assert = require('assert');
+
+Buffer.from(Array)
+var array_src = ([65, 66, 67]);
+var buff = Buffer.from(array_src,0,3);
+assert.equal(buff.toString(), "ABC");
+var buff2 = Buffer.from(array_src,0,-3);
+// Buffer.from(string,encoding)
+var string_utf = "ABC";
+var buff3 = Buffer.from(string_utf,'utf8');
```

Comment:
ditto.

Reply:
In the documentation is states that the `encoding` should be `hex`, but here it is `utf8` is this correct?